bom/application/pom.xml

```diff
@@ -209,6 +209,7 @@
         <log4j2-jboss-logmanager.version>1.0.0.Final</log4j2-jboss-logmanager.version>
         <log4j-jboss-logmanager.version>1.2.0.Final</log4j-jboss-logmanager.version>
         <avro.version>1.10.2</avro.version>
+        <apicurio-registry.version>2.0.0.Final</apicurio-registry.version>
```

Comment:
I don't see the dependency using it in the BOM file.

Reply:
Yea just force-pushed an update with this, otherwise the project wouldn't even build :-)